internal/zinc-scripted/src/test/scala/sbt/internal/inc/IncHandler.scala

```diff
@@ -829,6 +836,14 @@ case class ProjectStructure(
       }
       ()
     }
+
+  def assertShort(assertion: Boolean, message: => Any) = {
```

Comment:
nit: is this in use?

Reply:
Not any more.  I was leaving it for the next time, if that's ok - or I can remove it.